lib/service-worker-builder.js

```diff
@@ -85,7 +85,10 @@ module.exports = class ServiceWorkerBuilder {
 
   _rollupTree(tree, entryFile, destFile) {
     let rollupReplaceConfig = {
-      include: ['/**/ember-service-worker/**/*.js'],
+      include: [
+        '/**/ember-service-worker/**/*.js',
+        /^[a-z]:\/(?:[^\/:*?"<>|\r\n]+\/)*ember-service-worker\/(?:[^\/:*?"<>|\r\n]+\/)*[^\/:*?"<>|\r\n]*\.js$/i
```

Comment:
Could we add a comment with a link to the reference source and/or some sample paths that are matched? I'm always weary of long unexplained regular expressions in code.

Reply:
Sure thing we can. Will do soon-ish